jstests/aggregation/bugs/server7695_isodates.js

```diff
@@ -0,0 +1,268 @@
+// SERVER-7695: Add $isoWeek, $isoWeekYear, and $isoDayOfWeek aggregation expressions.
+
+(function() {
+    "use strict";
+    var coll = db.server7695;
+    var testOpCount = 0;
+
+    coll.drop();
+
+    // Seed collection so that the pipeline will execute.
+    assert.writeOK(coll.insert({}));
+
+    // Helper for testing that op returns expResult.
+    function testOp(op, value, expResult) {
+        testOpCount++;
+        var pipeline = [{$project: {_id: 0, result: {}}}];
+        pipeline[0].$project.result[op] = value;
+        var msg = "Exptected {"+op+": "+value+"} to equal: "+expResult;
+        assert.eq(coll.aggregate(pipeline).toArray()[0].result, expResult, msg);
+    }
+
+    // While development, there was a bug which caused an error with $dateToString if the order of
+    // %V and %G changed, so I added this test to prevent regression.
+    testOp('$dateToString', {date: new Date("1900-12-31T23:59:59Z"), format: "%V-%G"}, "01-1901");
```

Comment:
So I just made the tests less aggressive on windows and or 32-bit systems, like [here](https://github.com/mongodb/mongo/blob/r3.3.2/jstests/aggregation/bugs/server6189.js#L43-L47)